.vscode/tasks.json

```diff
@@ -0,0 +1,16 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=733558
+	// for the documentation about the tasks.json format
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "shell",
+			"label": "make: build",
+			"command": "make build",
```

Comment:
FWIW, this is really only used in CI in practice; local development almost always uses `make serve`, and having a default task run `make build` somewhat suggests it's a thing that's commonly done. I don't use these VS Code tasks very often myself, though -- is there an analogous one that might go along with `make serve`? Maybe something debug-related? 🤔

Reply:
Sorry for the delay, there's the concept of a default "build" task, and a default "test" task, and then one may have arbitrary tasks that you launch via the command bar. I don't think that "serve" would make most sense as a launch configuration of some sort.  This build task is harmless IMO but I'd be happy to remove it.